Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/connectmgr/persistance/dao/ConnectionManagerDAOBase.java

```diff
@@ -63,13 +65,19 @@ protected void saveBusinessDetail(BusinessDetail BusinessDetail, File file) {
             synchronized (file) {
                 JAXBContext context = JAXBContext.newInstance(BusinessDetail.class);
                 ObjectFactory factory = new ObjectFactory();
-                Marshaller marshaller = context.createMarshaller();
-                marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
+                DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();        
```

Comment:
You are creating a DocumentBuilderFactory here and setting the features. However while marshaling you are not using DocumentBuilderFactory in any way. How setting the features will help marshaling in this case?